recipes/python-simplemachinesforum/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "simplemachinesforum" %}
+{% set version = "0.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/simplemachinesforum-{{ version }}.tar.gz
+  sha256: 89699e282621b9c53ccf333149ec5a6fa51a101632a24eff69952cf2b320c5c5
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.0
+  run:
+    - python >=3.0
+    - requests
+    - bs4
+
+test:
+  imports:
+    - simplemachinesforum
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://about.me/oliver-zehentleitner
```

Comment:
This should be the URL to the project, not to your website. 

It might make sense to use the dev_url here.

Reply:
solved